Matter_Engine/Create/MaterialComposition.cs

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using BH.oM.Physical.Constructions;
+using BH.oM.Physical.Materials;
+
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+using BH.oM.Physical.Elements;
+using BH.oM.Geometry;
+using BH.oM.Dimensional;
+using BH.oM.Quantities.Attributes;
+using BH.Engine.Base.Objects;
+using BH.oM.Base;
+
+namespace BH.Engine.Matter
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Creates the combined MaterialComposition for a collection of IElementMs. Two Materials are considered equal if their names are equal.")]
+        [Input("elements", "The elements to get the combined materialCombination of.")]
+        [Output("materialComposition", "A material composition which contain which materials were used in the elements and in which proportions.")]
+        public static MaterialComposition CombinedMaterialComposition(IEnumerable<IElementM> elements)
```

Comment:
Starting reviewing - first comment.
Wondering if we should rename this method to simply `MaterialComposition`. The clarification it is "Combined" will come from the Descriptions and the change in arguments from the other Create methods.
I think this will be clearer as a "Create" - the other option would be to redefine as a Query or Compute. But think renaming would be my proposal.
@kThorsager @IsakNaslundBh thoughts?

Reply:
Yeah, Think just Create.MaterialComposition from a list of IElementM is clear enough (and agreed, with description).